packages/mobile/src/account/BankAccounts.tsx

```diff
@@ -73,24 +73,21 @@ function BankAccounts({ navigation, route }: Props) {
   }, [newPublicToken])
 
   function getBankDisplay(bank: BankAccount) {
+    // Todo: Consider adding a default placeholder image for banks without a logo available
```

Comment:
can you write an issue for this and remove the todo? ...or just do it? doesn't sound that hard

Reply:
So the hard part is I dont have a default image at hand not the act of adding the placeholder image itself. Also the percentage of bank accounts plaid does not return logo is really small so I don't feel it's a ticket-worthy task as all. I also think just the gray blank image doesn't look too bad.

Reply:
I think blank logo is fine